Dockerfile

```diff
@@ -1,9 +1,12 @@
-FROM node:14
+FROM node:14 as build-stage
 WORKDIR /dspfront
 COPY package*.json ./
-RUN npm i @vue/cli-service
-RUN npm install --production
-COPY . .
+RUN npm install
```

Comment:
Why not `--production`?

Reply:
It isn't necessary and requires @vue/cli-service to be installed. I used the [nginx section of the deployment guide](https://cli.vuejs.org/guide/deployment.html#docker-nginx) as a reference.